examples/warp_subscriptions/src/warp_subscriptions.rs

```diff
@@ -0,0 +1,181 @@
+//! `juniper_warp` subscriptions handler implementation.
```

Comment:
This might be better in `warp` behind  an `unstable_` or `non-standard_` flag?

Reply:
I'll add it under `subscriptions` feature flag to `juniper_warp` for now